### PR TITLE
Implement nickel_wifi action

### LINKED ISSUE #25: wifi toggle
I'd like an action to toggle Nickel's idea of wifi state.
I don't mean turning off the interface (ifconfig down) but actually disabling it (or I suppose enabling it) like the icon was tapped from the home screen and the slider moved...

I'd love to be able to get to this from the reading screen without popping all the way back to the home screen.

Would this be possible?  If so, my kobo's battery would thank you!!

### PR DESCRIPTION
**Pull request by geek1011**
closes #25

### CODE REVIEW

**Comment by geek1011** [comment]
I've tested this with:

```
menu_item:main:wifi autoconnect:nickel_wifi:autoconnect
menu_item:main:wifi autoconnect_silent:nickel_wifi:autoconnect_silent
menu_item:main:wifi enable:nickel_wifi:enable
menu_item:main:wifi disable:nickel_wifi:disable
menu_item:main:wifi toggle:nickel_wifi:toggle
menu_item:reader:wifi autoconnect:nickel_wifi:autoconnect
menu_item:reader:wifi autoconnect_silent:nickel_wifi:autoconnect_silent
menu_item:reader:wifi enable:nickel_wifi:enable
menu_item:reader:wifi disable:nickel_wifi:disable
menu_item:reader:wifi toggle:nickel_wifi:toggle
```

File: res/doc [hunk 1]
```diff
@@ -79,6 +80,12 @@
 #                                      reading_life:words        - My Words
 #                                      store:overdrive           - OverDrive (4.10.11655+) (note: if you don't have an active OverDrive account, it will give you a "Network Error")
 #                                      store:search              - Search
+#                   nickel_wifi    - one of:
+#                                      autoconnect        - attempts to enable and connect to wifi (similar to what happens when you open a link)
+#                                      autoconnect_silent - attempts to connect to wifi in the background (does nothing if wifi is disabled, the battery is low, is already connected, or there aren't any known networks in range) (no errors are shown) (similar to what happens when you turn on the Kobo)
+#                                      enable             - enables WiFi (but doesn't necessarily connect to it)
+#                                      disable            - disables WiFi (but doesn't necessarily connect to it)
```
Comment (NiLuJe) [inline res/doc:87]: The comment between parentheses doesn't make sense for disable ;).

File: res/doc [hunk 2]
```diff
@@ -79,6 +80,12 @@
...
+#                                      toggle             - disables WiFi (but doesn't necessarily connect to it)
```
Comment (NiLuJe) [inline res/doc:88]: In the same vein, disables ought to be toggles?

Comment (geek1011) [reply res/doc:88]: Oops... I need to abort that release I just started when you commented...

File: res/doc (hunk 1 above)
Comment (geek1011) [reply res/doc:87]: :facepalm:

Comment (geek1011) [reply res/doc:87]: Fixed in 1001be39aaf25f54edfd67b97103c0ab23ce59fd.

File: res/doc (hunk 2 above)
Comment (geek1011) [reply res/doc:88]: Fixed in 1001be39aaf25f54edfd67b97103c0ab23ce59fd.

**Pull request merged**